src/app/view/components/dot-persona-selector/dot-persona-selector.component.html

```diff
@@ -28,3 +28,47 @@
         ></dot-persona-selector-option>
     </ng-template>
 </dot-searchable-dropdown>
+
+<dot-dialog width="500px" *ngIf="addDialogVisible"
+    [header]="messagesKey['modes.persona.add.persona']"
+    [visible]="addDialogVisible"
+    [actions]="dialogActions"
+    (hide)="addDialogVisible = false">
+    <form [formGroup]="newPersonaForm">
```

Comment:
I guess this will be moved into a new component?  same with the functions in the TS file...

Reply:
This definitely needs to be it owns component, is the first step so it can be loaded dinamycally.